cvmfs/server/cvmfs_server_migrate.sh

```diff
@@ -473,6 +473,7 @@ _migrate_141() {
   fi
 
   echo "--> updating server.conf"
+  echo "CVMFS_USE_SSL_SYSTEM_CA=true" >> $server_conf
```

Comment:
It's unfortunately more difficult because r141 is already released.  We need to introduce a `_migrate_142` together with corresponding changes at the bottom of `cvmfs_server_migrate` and in `cvmfs_layout_revision()`